src/Env.php

```diff
@@ -37,6 +38,8 @@ public static function get($name)
     {
         if (self::$options & self::USE_ENV_ARRAY) {
             $value = isset($_ENV[$name]) ? $_ENV[$name] : false;
+        } elseif (self::$options & self::LOCAL_FIRST) {
+            $value = getenv($name, true) ?: getenv($name);
```

Comment:
Sorry again. The first `getenv` can return something evaluated as false (like `0`, `""`, etc). We need to check if the returned value is strictly `false`. In other words:

```php
$value = getenv($name, true);

if ($value === false) {
    $value = getenv($name);
}
```